src/components/Button/Button.js

```diff
@@ -0,0 +1,28 @@
+import PropTypes from "prop-types";
+import confirm from "../../assets/confirm.svg";
+import remove from "../../assets/remove.svg";
+
+export const Button = ({ text, icon, click, iconSize = 8 }) => {
+  const IconSize = `h-${iconSize} w-${iconSize}`;
+  return icon ? (
+    <div
+      onClick={click}
+      className="px-12 hover:scale-125 cursor-pointer transition-all "
+    >
+      <img
+        className={IconSize}
+        src={icon === "confirmed" ? confirm : remove}
+        alt="icon"
+      />
+    </div>
+  ) : (
+    <button onClick={click}>{text}</button>
```

Comment:
This doesn't really look like one component, but two. Simple Button only accepts `click` and `text`, whereas Icon Button accepts `icon`, `click` and `iconSize`. You also calculate a class for Icon Button only `IconSize`. The only common property is `click`. IMO, that should be 2 separate components. But for now, you can keep it.